aws/irsa.tf

```diff
@@ -1,12 +1,5 @@
 data "aws_caller_identity" "current" {}
 
-data "aws_region" "current" {}
-
-locals {
```

Comment:
is this no longer required?

Reply:
nope, they were unused according to tflint. Probably refactored them away earlier :)